ansible/roles/cartodb.postgresql/tasks/main.yml

```diff
@@ -23,6 +23,9 @@
  - name: Configure PostgreSQL host-based authentication
    template: src=pg_hba.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/pg_hba.conf
 
+ - name: Configure PostgreSQL conf
+   template: src=postgresql.conf.j2 dest=/etc/postgresql/{{ postgresql_version }}/main/postgresql.conf
+
```

Comment:
As an alternative, you may be able to setup service notifications for these templates and then use the following to trigger restarts for the queued handlers:

```yaml
- meta: flush_handlers
```